### ISSUE TITLE
type '_StreamHandlerTransformer<List<int>, List<int>>' is not a subtype of type 'StreamTransformer<Uint8List, List<int>>' of 'streamTransformer'

### ISSUE THREAD

**Issue by karnadii** [flutterchina/dio#357]
### New Issue Checklist

- [x] I have searched for a similar issue in the [project](https://github.com/flutterchina/dio/issues) and found none

### Issue Info

| Info                           | Value                                                 |      |
| ------------------------------ | ----------------------------------------------------- | ---- |
| Platform Name                  | flutter, android                          |      |
| Platform Version               | v1.8.1-pre.11                             |      |
| Dio Version                    | 2.1.8                                  |      |
| Android Studio / Xcode Version | e.g. Android Studio 3.3.2 / Xcode 10.2.1              |      |
| Repro rate                     | e.g. all the time (100%) / sometimes x% / only once   |      |
| Repro with our demo prj        | e.g. does it happen with our demo project?            |      |
| Demo project link              | e.g. link to a demo project that highlights the issue |      |

### Issue Description and Steps

I just upgrade my flutter using channel master to  v1.8.1-pre.11  and upgrade dio to 2.1.8
my code is exactly the same as example
```dart
import 'package:dio/dio.dart';
void getHttp() async {
  try {
    Response response = await Dio().get("http://www.google.com");
    print(response);
  } catch (e) {
    print(e);
  }
}
```

**Comment by onesmarter** [comment]
I had the same problem as you.

**Comment by calvinmuller** [comment]
Same for me 
```
 Unhandled Exception: DioError [DioErrorType.DEFAULT]: type '_StreamHandlerTransformer<List<int>, List<int>>' is not a subtype of type 'StreamTransformer<Uint8List, List<int>>' of 'streamTransformer'#0      Stream.transform (dart:async/stream.dart)
#1      DefaultTransformer.transformResponse (package:dio/src/transformer.dart:102:48)
```

**Comment by binary-zh** [comment]
升级之后我也遇到了一样的错误

**Comment by waitwalker** [comment]
我也遇到了

**Comment by Charles1198** [comment]
+1

**Comment by allencelee** [comment]
求官方赶紧修复 看来是所有人都遇到这个问题了

**Comment by hpoul** [comment]
seriously, would you guys mind stopping spamming this issue? there seems to be already a PR, so if you don't have anything useful to add just add your reaction or subscribe.. I subscribed so i get notified once there is any progress, not so that I get notified for everyone who has that issue as well..

**Comment by gerryhigh** [comment]
+1 as well for me--Dio is broken until a fix is released. Thanks.

**Comment by wendux** [comment]
please update to 2.1.9 and try again.

**Comment by arfaWong** [comment]
@wendux  upgrade to 2.1.9 and still had the same problem

**Comment by gerryhigh** [comment]
2.1.9 worked for me. @arfaWong  make sure you do a flutter clean

**Comment by CoderJava** [comment]
> 2.1.9 worked for me. @arfaWong make sure you do a flutter clean. Thanks @wendux

Yes. It's worked for me. Thanks.

**Comment by SalemCode8** [comment]
2.1.10 and this problem still present really !!

**Comment by siyasangab** [comment]
2.1.9 and I have the same issue

**Comment by jaripekkala** [comment]
I started receiving this error when changing to the `dev` flutter channel and stopped when changed back to `beta`.

**Comment by arfaWong** [comment]
2.1.13 worked for me. Thanks.
